Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -3,11 +3,10 @@ Copyright (c) 2018 Rohan Mitta. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Rohan Mitta, Kevin Buzzard, Alistair Tucker, Johannes Hölzl, Yury Kudryashov
 -/
-import Mathlib.Logic.Function.Iterate
+import Mathlib.Topology.Bornology.Hom
```

Comment:
Is there a reason that you moved this import to the top of the list?

Reply:
Now they're in alphabetical.
I understand import order is sometimes important, but when it's not, alphabetical makes the most sense to me.

Reply:
How is this alphabetical? Surely Bornology comes after Algebra?

Reply:
:facepalm: Indeed, reverted. Too early in the morning...